evap/evaluation/tests/test_commands.py

```diff
@@ -85,14 +85,15 @@ def test_dumpdata_called(self):
 
 @override_settings(REMIND_X_DAYS_AHEAD_OF_END_DATE=[0, 2])
 class TestSendRemindersCommand(TestCase):
+    now = datetime.datetime.now()
```

Comment:
can you decide on one import style (`import datetime` or `from datetime import datetime`) and apply that consistently everywhere? (i think i prefer the second style in this case because otherwise the code gets a lot of redundency)